medallion/backends/memory_backend.py

```diff
@@ -104,13 +107,19 @@ def _pop_expired_sessions(self):
             self.next.pop(item)
 
     def load_data_from_file(self, filename):
-        with open(filename, "r") as infile:
-            self.data = json.load(infile)
+        if isinstance(filename, string_types):
+            with io.open(filename, "r", encoding="utf-8") as infile:
```

Comment:
could we place the encoding in the function itself and pass it to io.open(). In the event someone wishes to use a different one?

Reply:
You mean like

```python
encoding="utf-8"
io.open(filename, "r", encoding=encoding)
```
where the encoding is in a variable?  I don't see what that accomplishes?  Maybe I don't understand the suggestion.

Reply:
No, I meant more like:

``` python
def load_data_from_file(self, filename, encoding="utf-8"):
    if isinstance(filename, string_types):
        with io.open(filename, "r", encoding=encoding) as infile:
```

Reply:
That was in my original [comment](https://github.com/oasis-open/cti-taxii-server/pull/99#issuecomment-587852116).  @clenk subsequently [decided](https://github.com/oasis-open/cti-taxii-server/pull/99#issuecomment-588288204) that if callers want to set the encoding themselves, they should create a file-like object and not pass a string filename.  That way, they can set the file-like object up however they want, including choosing an encoding.  I [agreed](https://github.com/oasis-open/cti-taxii-server/pull/99#issuecomment-588498330); it made sense to me.  So we decided not to have an `encoding` parameter to the function.